overlay/tun_notwin.go

```diff
@@ -0,0 +1,14 @@
+//go:build !windows
+// +build !windows
```

Comment:
Is there any harm in just putting this function in `overlay/tun.go`? I dont think we need a notwin file for it, it will still compile on windows just not be used.

Reply:
oh haha

https://github.com/slackhq/nebula/pull/903/commits/5159263448ee51bf26ceb05f4b737586fab833d5